.pre-commit-config.yaml

```diff
@@ -55,9 +48,3 @@ repos:
     rev: v1.7.7
     hooks:
       - id: actionlint
-
-  - repo: https://github.com/pycqa/isort
-    rev: 6.0.1
-    hooks:
-      - id: isort
-        name: isort (python)
```

Comment:
The goal is to have `ruff format` take care of `isort` rules, and the two conflict. So removing this.